concert/tests/integration/test_grating_interferometry_experiments.py

```diff
@@ -139,42 +140,45 @@ async def _test_stepping(self, stepping_type):
         if stepping_type not in ["reference", "object"]:
             raise Exception("Stepping type not known.")
 
-        self.assertEqual(len(self.acc.items[self.exp.get_acquisition(stepping_type + "_stepping")]),
+        items = (await self.acc.get_items(self.exp.get_acquisition(stepping_type + "_stepping")))
+        self.assertEqual(len(items),
                          (await self.exp.get_num_periods() *
                           await self.exp.get_num_steps_per_period()))
 
         stepping_start = await self.exp.get_stepping_start_position()
         step_size = await self.exp.get_grating_period() / await self.exp.get_num_steps_per_period()
-        for i in range(len(self.acc.items[self.exp.get_acquisition(stepping_type + "_stepping")])):
-            radio = self.acc.items[self.exp.get_acquisition(stepping_type + "_stepping")][i]
+        for i in range(len(items)):
+            radio = items[i]
             stepping_position = i * step_size + stepping_start
             self.assertAlmostEqual(radio[1, 0], stepping_position.to(q.um).magnitude, places=3)
             self.assertEqual(radio[0, 0], 1.0)
 
     async def test_reference_stepping(self):
         await self._test_stepping("reference")
-        for i in range(len(self.acc.items[self.exp.get_acquisition("reference_stepping")])):
-            radio = self.acc.items[self.exp.get_acquisition("reference_stepping")][i]
+        items = (await self.acc.get_items(self.exp.get_acquisition("reference_stepping")))
+        for i in range(len(items)):
+            radio = items[i]
             self.assertEqual(radio[0, 1], (await self.exp.get_flat_position()).to(q.mm).magnitude)
 
     async def test_object_stepping(self):
         await self._test_stepping("object")
-        for i in range(len(self.acc.items[self.exp.get_acquisition("object_stepping")])):
-            radio = self.acc.items[self.exp.get_acquisition("object_stepping")][i]
+        items = (await self.acc.get_items(self.exp.get_acquisition("object_stepping")))
+        for i in range(len(items)):
+            radio = items[i]
             self.assertEqual(radio[0, 1], (await self.exp.get_radio_position()).to(q.mm).magnitude)
 
     async def test_addon(self):
-        self.assertAlmostEqual(self.phase_stepping_addon.object_intensity[1, 1], 0.5, places=3,
+        self.assertAlmostEqual((await self.phase_stepping_addon.get_object_intensity())[1, 1], 0.5, places=3,
```

Comment:
This is only a class variable, not a property

Reply:
Ok, forget it. Github didn't show me the diff off Addons.py since there was changed to much...